dist/docker/scyllasetup.py

```diff
@@ -68,7 +68,7 @@ def io(self):
 
     def cqlshrc(self):
         home = os.environ['HOME']
-        hostname = subprocess.check_output(['hostname', '-i']).decode('ascii').strip()
+        hostname = self._listenAddress if self._listenAddress else subprocess.check_output(['hostname', '-i']).decode('ascii').strip()
```

Comment:
@asias please review

Reply:
The hostname is passed to cqlsh as the host to connect to, right? 

If the user specifies rpc_address, we use pass rpc_address instead of listen_address. 

```
    , listen_address(this, "listen_address", value_status::Used, "localhost",
        "The IP address or hostname that Scylla binds to for connecting to other Scylla nodes. You must change the default setting for multiple nodes to communicate. Do not set to 0.0.0.0, unless you have set broadcast_address to an address that other nodes can use to reach this node.")


    , rpc_address(this, "rpc_address", value_status::Used, "localhost",
        "The listen address for client connections (Thrift RPC service and native transport).Valid values are:\n"
        "\n"
        "\tunset:   Resolves the address using the hostname configuration of the node. If left unset, the hostname must resolve to the IP address of this node using /etc/hostname, /etc/hosts, or DNS.\n"
        "\t0.0.0.0 : Listens on all configured interfaces, but you must set the broadcast_rpc_address to a value other than 0.0.0.0.\n"
        "\tIP address\n"
        "\thostname\n"
        "Related information: Network\n")
```

So, the priority would be something like below:

rpc_address > listen_address -> hostname -i

Reply:
> The hostname is passed to cqlsh as the host to connect to, right?

Yes.

> So, the priority would be something like below:
> 
> rpc_address > listen_address -> hostname -i

Modified the code based on this priority.